.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: go
 go:
-  - 1.10.x
   - 1.11.x
+  - 1.12.x
```

Comment:
I guess we need to use 1.13 and 1.14 and remove 1.11 and 1.12.

Reply:
Since golang is compitable with old version, we only need test 1.11.x and 1.12.x because some of people will not follow the latest golang versions.